botocore/model.py

```diff
@@ -54,7 +54,7 @@ class Shape(object):
                         'xmlNamespace', 'resultWrapper', 'xmlAttribute',
                         'eventstream', 'event', 'eventheader', 'eventpayload',
                         'jsonvalue', 'timestampFormat', 'hostLabel']
-    METADATA_ATTRS = ['required', 'min', 'max', 'sensitive', 'enum',
+    METADATA_ATTRS = ['required', 'min', 'max', 'pattern', 'sensitive', 'enum',
```

Comment:
Would you mind also adding this to the docstring on L135-L142?

Reply:
done!